org-formation/300-account-defaults/bedrock-agent-role.yaml

```diff
@@ -31,7 +31,9 @@ Resources:
                 Action: "lambda:InvokeFunction"
                 Resource:
                   - !Sub "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:*"
-
+              - Effect: Allow
+                Action: "s3:*"
+                Resource: "*"
```

Comment:
why not use just apply the [AmazonS3FullAccess policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonS3FullAccess.html) managed policy?

Reply:
Good idea! Updated.